Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "kubewarden-policy-sdk"
 description = "Kubewarden Policy SDK for the Rust language"
 repository = "https://github.com/kubewarden/policy-sdk-rust"
-version = "0.2.3"
+version = "0.2.4"
```

Comment:
should we bump already to 0.2.5 for a new release?

Reply:
Oh yes, sorry... there's something I have to cleanup inside of Cargo toml too.

Reply:
fixed now